spec/lib/kiev/shoryuken_spec.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "kiev/test"
+
+if defined?(::Shoryuken)
+  RSpec.describe Kiev::Shoryuken do
+    # {
+    #   message_id: "437eb14c-7a98-457c-a431-01671564237e",
+    #   body: "test",
+    #   message_attributes: message_attributes,
+    #   delete: nil
+    # }
+    # {
+    #   "request_id"    => "acc6acfe-525e-49a6-a12b-ce3f07564620",
+    #   "request_depth" => 1,
+    #   "tree_path"     => "B"
+    # }
+
+    before(:all) do
+      # Shoryuken skips configure_server blocks unless it's running in a worker
+      # process. "worker process"ness is defined only by existence of this module
+      module Shoryuken::CLI
```

Comment:
Maybe you can simply include corresponding file here? `require "shoryuken/cli"`

Reply:
I do acknowledge that this looks extreme.
But no. It's defined in `bin`, so it's not on the load path. Besides, it contains non-trivial code that we don't need here anyway.

I thought of suggesting a change to Shoryuken to make it more testable, but I haven't agreed with myself on any particular approach yet.

Reply:
Shouldn't it be in https://github.com/blacklane/kiev/blob/master/spec/spec_helper.rb then?

Reply:
It's not used anywhere else yet, so no, it shouldn't.

Reply:
Well, `Que` job is also used only in a part related to `Que`. `spec_helper` is a right place for it.

Reply:
And I believe that's a mistake, unless there are specific technical advantages to it of which I'm not aware.